README.md

```diff
@@ -20,15 +20,15 @@
 
 Search UI allows you to quickly implement search experiences without re-inventing the wheel.
 
-It supports **[React](https://reactjs.org)** and **works with any search API**.
+It is built with **[React](https://reactjs.org)** and **works with any search API**.
```

Comment:
Hmmm the components are the only things that use React, right? I feel like saying `Built with React` here might scare people away who might want to build their own vanilla or other framework based components

Reply:
I don't know what it is about that that I don't like, it just doesn't sound right. "supports React", but it is React! 🤔. I'll revert my change, don't want to confuse people.

Reply:
"Includes React components"?

Reply:
I think I'm just overthining it.